### ISSUE TITLE
Disable TCP in 107

### ISSUE THREAD

**Issue by cpeterso** [mozilla-mobile/fenix#27487]
We want to disable TCP (`tcp-feature` and `tcp-cfr` flags in the `mr2022` Nimbus feature) in 107, so we can re-enable it in a Nimbus rollout in 107 Release. We want TCP to be enabled by default in 108+. Note that we don't want to disable TCP in all Beta versions, just 107 (which just happens to be in Beta right now). TCP is enabled by default in 108 and when 108 merges from Nightly to Beta, we want TCP to still be enabled by default in Beta 108 (and Release 108, etc).

https://github.com/mozilla-mobile/fenix/blob/releases_v107.0.0/nimbus.fml.yaml#L184-L208

@jhugman suggested we add a `default` block for beta and release in the v107 branch:

```
channel: beta, release
value:
  sections-enabled:
    tcp-feature: false
    tcp-cfr: false
```

---

In 107 Release, we will use a Nimbus rollout to gradually set `tcp-feature` = `true` for more users. We will also run a TCP experiment in parallel with the TCP rollout.

The experiment's control branch will enable TCP without the TCP CFR:

```
tcp-feature: true
tcp-cfr: false
```

And the experiment's treatment branch will enable both TCP and the TCP CFR:
```
tcp-feature: true
tcp-cfr: true
```

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FNXV2-21841)

**Comment by Mugurell** [comment]
Opened https://github.com/mozilla-mobile/fenix/pull/27516 to disable TCP in 107.

Regarding 
> @jhugman suggested we add a default block for beta and release in the v107 branch:

that default block would be useful to not need to disable TCP by default in each new beta / release with a new patch but as I understand the plan is for 108 to have TCP enabled by default so if we are adding that block now we will have to have another patch to remove the said block when we'll no need to disable TCP by default anymore.
If TCP will ride the trains starting with 108 by not adding that block now means we won't have to delete the block in the beta for 108.

**Comment by cpeterso** [comment]
Sounds good. We do expect to ship with TCP enabled by default in 108.

**Comment by cpeterso** [comment]
@Mugurell Can we close this issue? Has the TCP flag been disabled in Beta 107?

**Comment by Mugurell** [comment]
> @Mugurell Can we close this issue? Has the TCP flag been disabled in Beta 107?

Seems to me that the TCP is disabled in 107.0b4.
Will let QA double-check and close the issue.

**Comment by delia-pop** [comment]
Verified as fixed in Beta 107.0b4 with Xiaomi 12 Pro (Android 12). The TCP feature and TCP CFR are disabled in the current version.
![1666962406737](https://user-images.githubusercontent.com/89388888/198599131-7f16d6c5-1b54-406e-aeb9-d9f6194228cb.JPEG)
